vendor/src/github.com/docker/engine-api/types/container/config.go

```diff
@@ -5,6 +5,20 @@ import (
 	"github.com/docker/go-connections/nat"
 )
 
+// HealthConfig holds configuration settings for the HEALTHCHECK feature.
+// For durations, a value of -1 means to inherit the default value.
```

Comment:
Instead of the magic number, can the durations be pointers? `nil` Means inherit.